### PR TITLE
Support print SkyWalking context to logs

### PR DESCRIPTION
**Pull request by hailin0**
### Support print skywalking context to logs
- [x] Update the documentation to include this new feature.

- [ ] If this pull request closes/resolves/fixes an existing issue, replace the issue number. Closes #<issue number>.
- [x] Update the [`CHANGES` log](https://github.com/apache/skywalking/blob/master/CHANGES.md).

### CODE REVIEW

**Review by wu-sheng — DISMISSED** [review]
This looks cool. @dmsolr @EvanLjp Would you like to recheck?

File: apm-sniffer/apm-toolkit-activation/apm-toolkit-logback-1.x-activation/src/main/resources/skywalking-plugin.def [hunk 1]
```diff
@@ -15,9 +15,11 @@
...
 
 toolkit-logback=org.apache.skywalking.apm.toolkit.activation.log.logback.v1.x.LogbackPatternConverterActivation
+toolkit-logback=org.apache.skywalking.apm.toolkit.activation.log.logback.v1.x.LogbackSkywalkingContextPatternConverterActivation
 toolkit-logback=org.apache.skywalking.apm.toolkit.activation.log.logback.v1.x.mdc.MDCConverterActivation
 toolkit-logback=org.apache.skywalking.apm.toolkit.activation.log.logback.v1.x.async.AsyncAppenderBaseInstrumentation
 toolkit-logback=org.apache.skywalking.apm.toolkit.activation.log.logback.v1.x.async.LoggingEventInstrumentation
 toolkit-logback=org.apache.skywalking.apm.toolkit.activation.log.logback.v1.x.logstash.TcpSocketAppenderActivation
 toolkit-logback=org.apache.skywalking.apm.toolkit.activation.log.logback.v1.x.logstash.TraceIdJsonProviderActivation
+toolkit-logback=org.apache.skywalking.apm.toolkit.activation.log.logback.v1.x.logstash.SkywalkingContextJsonProviderActivation
 toolkit-logback=org.apache.skywalking.apm.toolkit.activation.log.logback.v1.x.log.GRPCLogAppenderActivation
```
Comment (dmsolr) [inline apm-sniffer/apm-toolkit-activation/apm-toolkit-logback-1.x-activation/src/main/resources/skywalking-plugin.def:25]: Losing `org.apache.skywalking.apm.toolkit.activation.log.logback.v1.x.mdc.PrintMDCSkywalkingContextInterceptor` ?

**Review by dmsolr — CHANGES_REQUESTED** [review]

File: apm-sniffer/apm-toolkit-activation/apm-toolkit-logback-1.x-activation/src/main/resources/skywalking-plugin.def (hunk 1 above)
Comment (kezhenxu94) [reply apm-sniffer/apm-toolkit-activation/apm-toolkit-logback-1.x-activation/src/main/resources/skywalking-plugin.def:25]: > Losing `org.apache.skywalking.apm.toolkit.activation.log.logback.v1.x.mdc.PrintMDCSkywalkingContextInterceptor` ?

I think only `*Instrumentation` and `*Activation` classes are needed in `*.def` file

**Review by kezhenxu94 — COMMENTED** [review]
This looks good to me except for the name `SkyWalking` is written mistakenly in all kinds, such as `sky-walking`, `skywalking`, `Skywalking`

File: apm-sniffer/apm-toolkit-activation/apm-toolkit-logback-1.x-activation/src/main/resources/skywalking-plugin.def (hunk 1 above)
Comment (dmsolr) [reply apm-sniffer/apm-toolkit-activation/apm-toolkit-logback-1.x-activation/src/main/resources/skywalking-plugin.def:25]: emmm..

**Review by dmsolr — APPROVED** [review]

**Comment by wu-sheng** [comment]
> This looks good to me except for the name `SkyWalking` is written mistakenly in all kinds, such as `sky-walking`, `skywalking`, `Skywalking`

Good catch. Didn't notice before. @hailin0 Please fix all and change them to`SkyWalking`, this is what we call ourselves in the ASF and public channel.

**Review by wu-sheng — APPROVED** [review]

**Review by wu-sheng — APPROVED** [review]

**Review by wu-sheng — APPROVED** [review]
PR is ready, sharing same test issue here, https://github.com/apache/skywalking/pull/6781#issuecomment-825310929

If you have something depending on this, I could merge now. @hailin0 Do you have next PR coming?

**Comment by hailin0** [comment]
> Do you have next PR coming?

 PR over and Test passed

**Pull request merged**